webpack.config.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const HTMLPlugin = require('html-webpack-plugin');
+const ExtractPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  devtool: 'eval',
+  entry: `${__dirname}/app/entry.js`,
+  output: {
+    filename: 'bundle-[hash].js',
+    path: `${__dirname}/build`,
+  },
+  plugins: [
+    new HTMLPlugin({
+      template: `${__dirname}/app/index.html`,
+    }),
+    new ExtractPlugin('bundle-[hash].css'),
+  ],
+  module: {
+
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+      },
+      {
+        test: /\.scss$/,
```

Comment:
So per the lab instructions, you're actually supposed to refractor your webpack config, with the given code block. So I'll have to take a point for that.